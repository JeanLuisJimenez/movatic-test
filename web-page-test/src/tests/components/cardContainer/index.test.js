import CardContainer from '../../../components/cardContainer'
import { render } from '@testing-library/react'

it('CardContainer', () => {
    const renderedCard = render(<CardContainer title={'TEST TITLE'} cards={[{title: "Card Title 1", value: "Yes"}, {title: "Card Title 2", value: "No"}]}/>)

    expect(renderedCard).toMatchSnapshot()
})
