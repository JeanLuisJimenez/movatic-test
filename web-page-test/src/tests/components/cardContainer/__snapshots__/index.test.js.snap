// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardContainer 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="card-container"
      >
        <h2
          class="card-container-title"
        >
          TEST TITLE
        </h2>
        <div
          class="card-container-content"
        >
          <div
            class="card"
          >
            <h4
              class="card-title"
            >
              Card Title 1
            </h4>
            <h5
              class="card-value"
            >
              Yes
            </h5>
          </div>
          <div
            class="card"
          >
            <h4
              class="card-title"
            >
              Card Title 2
            </h4>
            <h5
              class="card-value"
            >
              No
            </h5>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="card-container"
    >
      <h2
        class="card-container-title"
      >
        TEST TITLE
      </h2>
      <div
        class="card-container-content"
      >
        <div
          class="card"
        >
          <h4
            class="card-title"
          >
            Card Title 1
          </h4>
          <h5
            class="card-value"
          >
            Yes
          </h5>
        </div>
        <div
          class="card"
        >
          <h4
            class="card-title"
          >
            Card Title 2
          </h4>
          <h5
            class="card-value"
          >
            No
          </h5>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
